visualize: True
###dkan
# aspect_th: 6.0 #width/hight
# whiteness_th: 0.9
# texture_th: 50
# smoothness_th: 0.85

###tsukuba
aspect_th: 3.0 #width/hight
whiteness_th: 0.90
texture_th: 5.0
smoothness_th: 0.75

hz: 10 #hz

###jikki
trans_upper_left: [202,30]
trans_upper_right: [455,29]
###tsukubag
# trans_upper_left: [271,50]
# trans_upper_right: [452,47]

###(100 cm ~ 30 cm)
###(30, 55, 83, 115, 155, 203, 265, 347)
stop_level: 203 #cell(v)

line_grad_th: 1.0 #y/x
line_length_th: 50 #cell(u)
close_line_th: 20 #cell

hsv_lo: [0,0,180]
hsv_hi: [180,50,255]

split_num: 10
resize_num: 70
